riak_test/yz_rt.erl

```diff
@@ -615,3 +615,6 @@ load_intercept_code(Node) ->
                               "*.erl"]),
     rt_intercept:load_code(Node, [CodePath]).
 
+-spec rpc_call_cluster([node()], Module :: atom(), Function :: atom(), [term()]) -> [term()].
+rpc_call_cluster(Cluster, Module, Function, Args) ->
+    [rpc:call(Node, Module, Function, Args) || Node <- Cluster].
```

Comment:
Good catch.  I suspect we should be using riak_core_util:multi_rpc, but I don't think we have riak_core in riak_test's deps.